Index: creader/TheBookshelf/apps/setup.py
===================================================================
diff --git a/creader/TheBookshelf/apps/setup.py b/creader/TheBookshelf/apps/setup.py
new file mode 100644
--- /dev/null
+++ b/creader/TheBookshelf/apps/setup.py
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+setup(
+    name='Creader-backend',
+    version='',
+    packages=['user', 'user.migrations', 'comment', 'comment.migrations', 'payment', 'payment.migrations', 'product',
+              'product.migrations', 'bookcase', 'bookcase.migrations', 'bookitem', 'bookitem.migrations',
+              'transactions', 'transactions.migrations'],
+    package_dir={'': 'creader/TheBookshelf/apps'},
+    url='',
+    license='',
+    author='tim',
+    author_email='',
+    description=''
+)
